libBIDS.sh

```
@@ -344,6 +344,53 @@ libBIDSsh_extension_json_rows_to_column_json_path() {
 }
 
+_libBIDSsh_load_custom_entities() {
+  # Load custom entities from JSON configuration files
+  # JSON files should be placed in ./custom directory
+  # Each JSON file should contain an "entities" array with objects having:
+  #   - name: entity short name
+  #   - display_name: entity display name for CSV headers
+  #   - pattern: bash glob pattern for matching
+
+  
+  local script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)